ci/docker/android-builder/Dockerfile

```diff
@@ -61,6 +45,23 @@ RUN mkdir $HOME/.android && \
     "extras;google;google_play_services" \
     "extras;google;m2repository" && \
   chmod -R o+rwX ${ANDROID_HOME}
+
+# --------------- Gradle Profiler -----------------
```

Comment:
This layer is more stable but it's cheap. 
I want to show that it requires Android SDK to build the project.